src/shared/views/map/MapTab/MapTab.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+import styles from './mapTab.css'
+
+const MapTab = (props) => {
+  return (
+    <img className={styles.mapImage} alt={`map-level-${props.index}`} src={`/static/maps/Level${props.index + 1}.jpg`} />
```

Comment:
The full URLS should be passed tot he component. This component is becoming too specific.